arbitrum-docs/stylus/reference/local-stylus-dev-node.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: 'How to run a local dev node with Stylus support'
+sidebar_label: 'Local Stylus dev node'
+description: 'A guide to run an Arbitrum local dev node with Stylus support on your local machine'
+author: jose-franco
+sme: jose-franco
+target_audience: 'Developers deploying smart contracts using Stylus'
+content-type: how-to
+sidebar_position: 4
+---
+
+import PublicPreviewBannerPartial from '../partials/_stylus-public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+This how-to guides you through the process of setting up a local Arbitrum Nitro dev node, with Stylus support, to help you deploy and test smart contracts written in Solidity or any of the [Stylus supported languages](/stylus/stylus-gentle-introduction.md#in-a-nutshell) in a fully controlled environment. By following these steps, you'll deploy and run a full development environment on your local machine that includes a Nitro dev node (L2), a dev-mode geth parent chain (L1), and multiple instances with different roles.
```

Comment:
```suggestion
This how-to guides you through the process of setting up a local Arbitrum Nitro dev node, with Stylus support, to help you deploy and test smart contracts written in Solidity or any [languages supported by Stylus](/stylus/stylus-gentle-introduction.md#in-a-nutshell) in a fully controlled environment. By following these steps, you'll deploy and run a full development environment on your local machine that includes a Nitro dev node (L2), a dev-mode geth parent chain (L1), and multiple instances with different roles.
```

Small suggestion to imply that even if there is no "official" support for a language via an SDK, users can still deploy programs in others